cmd/create.go

```diff
@@ -32,6 +32,59 @@ var (
 	})
 )
 
+func isGolang(dir string) bool {
+	if _, err := os.Stat(filepath.Join(dir, "go.mod")); err == nil {
+		return true
+	}
+	return false
+}
+
+func isNodejs(dir string) bool {
+	if _, err := os.Stat(filepath.Join(dir, "package.json")); err == nil {
+		return true
+	}
+	return false
+}
+
+func isPython(dir string) bool {
+	if _, err := os.Stat(filepath.Join(dir, "requirements.txt")); err == nil {
+		return true
+	}
+	// poetry and others
+	if _, err := os.Stat(filepath.Join(dir, "pyproject.toml")); err == nil {
+		return true
+	}
+	return false
+}
+
+func isDocker(dir string) bool {
+	if _, err := os.Stat(filepath.Join(dir, "Dockerfile")); err == nil {
```

Comment:
Idk if we should do this quite yet. Today, we don't support building from a dockerfile (yet). The docker type is only for pre-baked images.

Reply:
Although I guess this doesn't hurt anything. If they have a dockerfile, maybe they have an image.